src/server.js

```diff
@@ -91,6 +97,7 @@ function index(req, res, next) {
 			config
 		);
 		var template = _.template(file);
+		res.setHeader("Content-Security-Policy", "default-src *; style-src * 'unsafe-inline'; script-src 'self'; reflected-xss block");
```

Comment:
I'm not sure there's any need to send CSP on all requests (all other requests besides index.html would be static content).

> I'm not sure I fully understand reflected-xss block

> reflected-xss : Instructs a user agent to activate or deactivate any heuristics used to filter or block reflected cross-site scripting attacks, equivalent to the effects of the non-standard X-XSS-Protection header.

I've also didn't set extra hardened directives because I don't know what we would want to allow for eventual plugins to do.


> `*` is the default value for everything so we don't need to specify it for default-src

Explicitness. We could eventually set it to `self` instead, and configure other directives as needed.